storage/api/Storage.Samples.Tests/SetClientEndpointTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Xunit;
+public class SetClientEndpointTest
+{
+    [Fact]
+	public void SetEndpoint()
+    {
+        String projectId = "projectId";
+        String endpoint = "https://storage.googleapis.com";
+        var storageClient=SetClientEndpoint.SetEndpoint(projectId, endpoint);
+        Assert.Equal(endpoint,storageClient.Service.BaseUri);
+    }

```

Comment:
Let's use a dummy endpoint and not the default one, else we are never certain that the sample is doing as intended.
Also, you'll have to create a an instance of the sample class to call the sample method. We don't create static sample methods.

```suggestion
    public void SetsClientEndpoint()
    {
        string endpoint = "https://dummy.storage.googleapis.com";
        var storageClient = SetClientEndpoint.SetEndpoint(endpoint);
        Assert.Equal(endpoint, storageClient.Service.BaseUri);
    }

```